### DISCUSSION TITLE
Calico will be at KubeCon Europe 2024 March 19-23 in Paris

### DISCUSSION THREAD

**Discussion by frozenprocess** [projectcalico/calico#8520] [Announcements]
### TL/DR
KubeCon Paris can't wait to see you there!
March 19 - CalicoCon ( [registration required](https://link.tigera.io/uguPT 
))
March 20 - March 22 - Booth K17

**Hey everyone,**
KubeCon Europe 2024 train is approaching us and we wanted to make sure that you know where to find us!
Make sure to drop by our booth (K17) at the event and collect your Calico swags, we are also planning to have our core engineers available at the booth so make sure to bring your questions.

### Wait there is more!!
As the creator of Project Calico, we are excited to announce that CalicoCon is coming to KubeCon + CloudNativeCon Europe 2024 on March 19 in Paris! 
Don’t miss this opportunity to learn best practices for securing, observing, and troubleshooting cloud-native applications. Join us for workshops, keynotes, and networking opportunities.
Limited spots are available, so register now to secure your spot: https://link.tigera.io/uguPT
